ui/apos/components/AposExportModal.vue

```diff
@@ -288,23 +288,19 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+/* stylelint-disable declaration-property-unit-allowed-list */
+/* stylelint-disable time-min-milliseconds */
```

Comment:
Do we need that for `inset: 0`?

Reply:
I haven't touch any style, everything is auto-repaired by stylelint.

Reply:
Stylelint added disable comments by itself? 🤔

Reply:
Found another way to fix it.